torch/_dynamo/utils.py

```diff
@@ -866,6 +866,7 @@ class CompilationMetrics:
     joint_graph_pass_time_us: Optional[int] = None
     log_format_version: int = LOG_FORMAT_VERSION
     inductor_config: Optional[str] = None
+    feature_usage: Optional[dict[str, bool]] = None
```

Comment:
Will they always be bools though? Will we log JKs that return ints the same way?

Reply:
If you feel strongly we can change the typing to allow anything, but in general we're logging usage, not the JK results (i.e. not only did we read the JK, but we actually used the feature).